### PR TITLE
12.0 mig base user role

### PR DESCRIPTION
**Pull request by novawish**
Credits to @jcdrubay

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you check Travis and runbot status?

**Comment by novawish** [comment]
I fixed runbot, however i don't know the reason why travis is failed, do you have any idea?
image

Thanks

**Comment by sebalix** [comment]
We should upgrade to PostgreSQL 9.5 (currently the 9.3 is used), `ON CONFLICT` syntax seems to be available only on 9.5. Try to change the version in `.travis.yml` file.

**Comment by novawish** [comment]
@sebalix Thank you very much, i updated pg version in `.travis.yml` to 9.5
@pedrobaeza Runbot and travis is ok now

File: base_user_role/__manifest__.py [hunk 1]
```diff
@@ -3,11 +3,11 @@
 
 {
     'name': 'User roles',
-    'version': '11.0.1.0.1',
+    'version': '12.0.1.0.1',
```
Comment (sebalix) [inline base_user_role/__manifest__.py]: Reset to `12.0.1.0.0`.

File: base_user_role/__manifest__.py [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
     'category': 'Tools',
     'author': 'ABF OSIELL, Odoo Community Association (OCA)',
     'license': 'AGPL-3',
-    'maintainer': 'ABF OSIELL',
+    'maintainers': ['ABF OSIELL, jcdrubay'],
```
Comment (sebalix) [inline base_user_role/__manifest__.py]: Some quotes are missing I think.

**Review by sebalix — CHANGES_REQUESTED** [review]

**Review by sebalix — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]
Functional review :+1: 

Is it ready to merge?

THX!

**Pull request merged**